src/firmware_new/boards/frankie_v1/io/proto_multicast_communication.h

```diff
@@ -0,0 +1,143 @@
+#pragma once
+
+#include <pb.h>
+
+typedef struct ProtoMulticastCommunicationProfile ProtoMulticastCommunicationProfile_t;
+
+/**
+ * Event Flags: used to signal tasks to unblock and run specific actions
+ */
```

Comment:
I'd prefer if Event Flags were clarified here quickly instead of just letting the reader figure out its arbitrary.

Reply:
these flag masks are chosen by us, I updated the comment to say they are masks 

what do you think would help in this case?